rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -133,6 +136,9 @@
     private FastBridgeFederationInformation fastBridgeFederationInformationsToSave = null;
     private long receiveHeadersLastTimestamp = 0;
 
+    private Long utxoExpirationCheckpointRskBlockHeight;
+    private Long utxoExpirationCheckpointTimestamp = 0L;
```

Comment:
```suggestion
    private Long nextUtxoExpirationCheckpointHeight;
    private Long nextUtxoExpirationCheckpointTimestamp;
```
We shouldn't initialize these variables if we are checking for null value in the getter

Reply:
Resolved